PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -56,8 +56,8 @@ function TouchJump:Create(parentFrame)
 	JumpButton.ImageRectOffset = Vector2.new(176, 222)
 	JumpButton.ImageRectSize = Vector2.new(174, 174)
 	JumpButton.Size = UDim2.new(0, jumpButtonSize, 0, jumpButtonSize)
-	JumpButton.Position = isSmallScreen and UDim2.new(1, jumpButtonSize * -2.25, 1, -jumpButtonSize - 20) or
-		UDim2.new(1, jumpButtonSize * -2.75, 1, -jumpButtonSize - 120)
+	JumpButton.Position = isSmallScreen and UDim2.new(1, -(jumpButtonSize*1.5-10), 1, -jumpButtonSize - 20) or
```

Comment:
I believe the key is to make sure the flag starts with 'User'

On Mon, Aug 8, 2016 at 5:59 PM, Ben Tkacheff <ben@roblox.com> wrote:

> We have in the past, this change shows a removal of a flag:
> https://github.com/ROBLOX/Core-Scripts/commit/
> cd4c3031742f084332499f09207e8b2e5e7af126
>
> On Mon, Aug 8, 2016 at 5:56 PM, garnold24 <notifications@github.com>
> wrote:
>
>> In PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayer
>> Scripts/ControlScript/MasterControl/TouchJump.rbxmx
>> <https://github.com/ROBLOX/Core-Scripts/pull/491#discussion_r73983121>:
>>
>> > @@ -56,8 +56,8 @@ function TouchJump:Create(parentFrame)
>> >  	JumpButton.ImageRectOffset = Vector2.new(176, 222)
>> >  	JumpButton.ImageRectSize = Vector2.new(174, 174)
>> >  	JumpButton.Size = UDim2.new(0, jumpButtonSize, 0, jumpButtonSize)
>> > -	JumpButton.Position = isSmallScreen and UDim2.new(1, jumpButtonSize * -2.25, 1, -jumpButtonSize - 20) or
>> > -		UDim2.new(1, jumpButtonSize * -2.75, 1, -jumpButtonSize - 120)
>> > +	JumpButton.Position = isSmallScreen and UDim2.new(1, -(jumpButtonSize*1.5-10), 1, -jumpButtonSize - 20) or
>>
>> I didn't think you could flag player scripts?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ROBLOX/Core-Scripts/pull/491/files/3c8b3849bfa4985880ebcb4a80f45e3e0302c39e#r73983121>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AG9VilpgDDkNwLOohj3z0k2wzBGJkQOMks5qd9BVgaJpZM4JfnSG>
>> .
>>
>
>
>
> --
> ben
>



-- 
ben